apps/extension/src/wasm-build-action.ts

```diff
@@ -43,7 +42,7 @@ async function executeWorker(
   witness: WitnessData,
   fullViewingKey: string,
   actionPlanIndex: number,
-): Promise<Jsonified<Action>> {
+): Promise<JsonValue> {
```

Comment:
Same Q here — don't we want `Jsonified<Action>`?

Reply:
we could probably assert that, but at the return statement we're calling a function that really does return `JsonValue` explicitly